### DISCUSSION TITLE
An error occured while running the seed command:

### DISCUSSION THREAD

**Discussion by KakoozaJerry** [prisma/prisma#10282] [Q&A]
```

  3 const prisma = new PrismaClient()
  4 
  5 async function main(){
→ 6     const newUser = await prisma.user.create(
  Error occurred during query execution:
ConnectorError(ConnectorError { user_facing_error: None, kind: RawError { code: "unknown", message: "Command failed (IllegalOperation): This MongoDB deployment does not support retryable writes. Please add retryWrites=false to your connection string.)" } })
    at cb (/Users/jerry/Documents/WorkingSpace/TTS/nextJS/auth0-app/node_modules/@prisma/client/runtime/index.js:38506:17)
    at async main (/Users/jerry/Documents/WorkingSpace/TTS/nextJS/auth0-app/prisma/seed.js:6:21) {
  clientVersion: '3.4.2'
}

An error occured while running the seed command:
Error: Command failed with exit code 1: node prisma/seed.js
```

**Comment by austincrim** [comment]
Hey @KakoozaJerry 👋!

Could you provide a little more info about your environment? Are you trying to connect to a local instance of Mongo?

**Comment by KakoozaJerry** [comment]
Yes , I am trying to connect to a local mongo db instance

**Reply by austincrim** [reply]
Currently, in order to use Prisma with MongoDB, your Mongo instance **has to have a replica set deployment** as well. You can [set this up locally](https://docs.mongodb.com/manual/tutorial/deploy-replica-set/), but we recommend using Mongo Atlas as the easiest way to get going.

From our [docs](https://www.prisma.io/docs/concepts/database-connectors/mongodb#example): 
> The MongoDB database connector uses transactions to support nested writes. Transactions requires a [replica set](https://docs.mongodb.com/manual/tutorial/deploy-replica-set/) deployment. The easiest way to deploy a replica set is with [Atlas](https://docs.atlas.mongodb.com/getting-started/). It's free to get started.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
